### ISSUE TITLE
Promise: Home page - explore section

### ISSUE THREAD

**Issue by mixth** [wevisdemo/parliament-watch#100]
The PR shall be merged into `promise/main` branch.

This issue is blocked by:
- #96 
- #99 

# Purpose
To filter and show promises by statuses/parties/categories/keywords

# Note
- The page will filter promises based on these filters and feed the result to List component implemented in #96.
- Tap "ดูทั้งหมด" will remove the `cappedAt` of the component and show all available promises

# UI
Please refer to [latest UI in Figma](https://www.figma.com/design/ywiKyd0XIgDXf8crX4aoYE/Design_Phase1-2-%5BPromise-Tracker%5D?node-id=4552-9015&node-type=frame&t=HmpwBOeDR4PxrdVq-0) from the design team.

![Frame 1414](https://github.com/user-attachments/assets/4907f1e9-4eab-4366-ac47-2b0c1dd58976)

**Comment by Th1nkK1D** [comment]
Merged #111
